website/content/v1.0/guides/deploy-metrics-server.md

```diff
@@ -30,7 +30,7 @@ We can have Kubelet Serving Certificate Approver and metrics-server installed on
 cluster:
   extraManifests:
     - https://raw.githubusercontent.com/alex1989hu/kubelet-serving-cert-approver/main/deploy/standalone-install.yaml
-    - https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/components.yaml
+    - https://github.com/kubernetes-sigs/metrics-server/releases/{{< release >}}/download/components.yaml
```

Comment:
would this work the k8s sigs repo?

Reply:
oops... too much search/replace, thanks for noticing that